alvr/client_core/src/platform/android.rs

```diff
@@ -340,6 +340,7 @@ pub fn video_decoder_split(
             format.set_str("mime", mime);
             format.set_i32("width", 512);
```

Comment:
Is it possible to somehow get foveated resolution at this point to give a proper hint to the decoder?

Reply:
Yes, we have it, but it doesn't matter, the csd0 packet already contains the resolution info.